lang/cs/Org.Apache.REEF.Utilities/Yarn.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+using Org.Apache.REEF.Utilities.Diagnostics;
+using Org.Apache.REEF.Utilities.Logging;
+
+namespace Org.Apache.REEF.Utilities
+{
+    /// <summary>
+    /// A static helper class for the YARN runtime.
+    /// </summary>
+    public static class Yarn
+    {
+        private static readonly Logger Logger = Logger.GetLogger(typeof(Yarn));
+
+        /// <summary>
+        /// The Hadoop configuration directory environment variable.
+        /// </summary>
+        public const string HadoopConfDirEnvVariable = "HADOOP_CONF_DIR";
```

Comment:
The `const` strings are dangerous when their value ever changes, right? I believe the C# compiler is free to inline them in client code of this. Maybe make them `readonly` instead?